client/src/Pages/Login/IsAuthenticated.jsx

```diff
@@ -0,0 +1,25 @@
+import { CircularProgress } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { useAuthContext } from "../../api/useAuthContext";
+import {useEffect} from "react";
+export const IsAuthenticated = () => {
+  const navigate = useNavigate();
+  const context = useAuthContext();
+
+  useEffect(() => {
+    if (context?.user?.isAuthenticated) {
+      console.log("User is authenticated, redirecting to overview..");
```

Comment:
Denne log meldingen burde ha med noe info om user og vi burde ikke logge når noe går som forventet, vi logger feil. Så foreslår at den blir flyttet ned til else delen av koden.

Reply:
Dette var nok for testing purposes fra min side, det kan fjernes